newsfragments/2612.misc.rst

```diff
@@ -0,0 +1 @@
+Add support for the new Umbral API
```

Comment:
Perhaps include a version number here for posterity.

Reply:
Fixed